### DISCUSSION TITLE
missing desktop entry for epoptes-client

### DISCUSSION THREAD

**Discussion by freddii** [epoptes/epoptes#132] [General]
there is a desktop entry for the client at:
https://github.com/epoptes/epoptes/blob/master/epoptes-client/epoptes-client.desktop

but my debian package epoptes-client_21.01-1_all.deb does not have a desktop file at all.

**Comment by alkisg** [comment]
Hi freddii,

```shell
$ dpkg -L epoptes-client | grep desktop
/etc/xdg/autostart/epoptes-client.desktop
```

It goes there in order to autostart when the users log in. It doesn't belong in the users' main menu.

**Reply by freddii** [reply]
thanks for the explanation !
